src/components/shmoke.vue

```diff
@@ -0,0 +1,93 @@
+<template>
+<div id='s' class="fl ma2 tc b--solid bg-light-green shadow-5 db pa3"> 
+        <h2>🌲 SHMOKE up FELLA 🌲 </h2>
+        <img class="snoop" @click="shmoke_weed"  :src = getimageUrl() />
+</div>     
+
+</template>
+
+
+<script>
+import { METHODS } from 'http';
+import eventBus from '../eventBus';
+export default {
+    data(){
+        return {
+            shmoking: [
+                {
+            shmoked:true, 
+            name:"shmokie",
+            run: () => {
+                eventBus.$emit('achievement-unlocked' , 'We only smoke quads')
+                eventBus.$emit('big-kush-energy')
+                this.$modal.show('dialog', {
+              title: `<h1> SHMOKING GANJA BIG BOI , ENJOY A 30 🍆 BOOST </h1>`,
+              buttons:[{
+                  title: "puff puff 🅱IGGA"
+              }]
+                }) 
+            }
+                }
+            ]
+        }
+    },
+    computed:{
+        shmoke(){
+           return this.shmoking
+        }
+    },
+    methods:{
+        getsoundUrl(){
+            return require('../assets/items/shmoke.mp3');
+        },
+        getimageUrl() {
+            return require('../assets/items/tenor.gif')
+        },
+
+        shmoke_weed() {
+            this.shmoke[0].run()
+            var smoke_screem = document.getElementsByClassName('toking')
+            var shmokie = document.getElementsByClassName('swed')
+            var promise = shmokie[0].play();
+
+            if (promise !== undefined) {
+                    promise.then(function() {
+                    // Automatic playback started!
+                }).catch(function(error) {
+                    console.log(error)
+            });
```

Comment:
Legit question what's going on here lol? 🤔

Reply:
I was trying to get a sound to start playing when someone clicked on the shmoke ,(it was "SMOKE EVERY DAY" if youre wondering) and yeah it was throwing exceptions and i just wanted to catch it . (couldnt get it to work so i guess illl just take it out)